packages/next-auth/src/providers/boxyhq-saml.ts

```diff
@@ -0,0 +1,37 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface SAMLJacksonProfile {
+  id: string;
+  email?: string;
+  firstName?: string | null;
+  lastName?: string | null;
```

Comment:
```suggestion
  firstName?: string | null
  lastName?: string | null
```

Are these really optional, or guaranteed to return at least `null`, when missing on your side? Should we drop either the `?` or `null`?

Reply:
We can drop the null.